### ISSUE TITLE
REFPROP DQ input

### ISSUE THREAD

**Issue by thorade** [CoolProp/CoolProp#1845]
The following snippet works (taken from #1556):
```
import CoolProp as CP
vapState = CP.AbstractState("HEOS", "CO2")
vapState.update(CP.DmassQ_INPUTS, 15.0, 1.0)
print(vapState.T())
```
But the same with REFPROP as backend fails:
```
import CoolProp as CP
vapState = CP.AbstractState("REFPROP", "CO2")
vapState.update(CP.DmassQ_INPUTS, 15.0, 1.0)
print(vapState.T())
```
```
ValueError: This pair of inputs [DmassQ_INPUTS] is not yet supported
```

According to https://github.com/usnistgov/REFPROP-issues/issues/189 this input should be supported in RefProp as well, so it is only the wrapper that is missing.

**Comment by ibell** [comment]
Good idea!  This is a nice introductory modification.  I think you have the
chops to make the change yourself in CoolProp

**Comment by thorade** [comment]
Yes, would be great to finally contribute some code! Even if it will probably only be 2-10 lines...

**Comment by thorade** [comment]
Step 1: This message comes from this line:
https://github.com/CoolProp/CoolProp/blob/84dcbda935e2116e6316bf7b440919abc3321a65/src/Backends/REFPROP/REFPROPMixtureBackend.cpp#L1830
with all supported input pairs handled above it.

**Comment by xjvheule** [comment]
Has there been a solution found to this error or was this fixed in a later update because i also ran into this issue with REFPROP version 10.0.
